src/scss/colors.scss

```diff
@@ -2,20 +2,33 @@ $colors-enum: (
   primary: #7c46fb,
 );
 $shades-enum: (5, 10, 14, 20, 50);
+$alphas-enum: (0.18, 0.2, 0.24, 0.35, 0.5, 0.56, 0.65, 0.8);
 
 :root {
   --primary-color-deprecated: #f6e652;
+
   --primary-content--under: #000000;
-  --primary-content--under-rgb: 0, 0, 0;
   --primary-content--over: #ffffff;
+  --primary-accent--over: #ffffff;
+
+  // TODO: add util function from hex to rgb
```

Comment:
No need for comment. If function needed, most likely SCSS already provides it with https://sass-lang.com/documentation/modules/color . Normally, though, SCSS can use both intermittently, so may not even need one. 

```suggestion
```

Reply:
Got it. I've added the util function via this module https://github.com/sparkletown/sparkle/pull/2343/files#diff-adb1222623d1ae146d1b87f9ccb2f5cc097d12e1cc51c065b8c4ccc00029f61dR1

It looks like SCSS can use both intermittently and `rgba` function needs RBG format only (can't find the way to calculate its value via var CSS)

Reply:
URL https://github.com/sparkletown/sparkle/pull/2343/files#diff-adb1222623d1ae146d1b87f9ccb2f5cc097d12e1cc51c065b8c4ccc00029f61dR1 is entire diff of 85 files. What module?


> (can't find the way to calculate its value via var CSS)

This is the reason for SCSS existence, it pre-calculates values so that CSS doesn't have to.

Reply:
@goran-peoski-work and it loads the `colors.scss` file later 

<img width="1440" alt="Screenshot 2021-10-04 at 13 55 24" src="https://user-images.githubusercontent.com/26366773/135839573-4b1be4eb-cf41-47ec-8502-802b242085fd.png">